ckanext/depositar_theme/fanstatic/scss/index/index.scss

```
@@ -1,15 +1,17 @@
-@import 'variables.scss';
-@import 'masthead.scss';
-@import 'flash.scss';
-@import 'banner.scss';
-@import 'intro.scss';
-@import 'features.scss';
-@import 'showcases.scss';
-@import 'guide.scss';
-@import 'topics.scss';
-@import 'stats.scss';
-@import 'faq.scss';
-@import 'footer.scss';
+// Bootstrap
+@import "../../../../../../ckan/ckan/public/base/scss/bootstrap";
+@import 'variables';
+@import 'masthead';
+@import 'flash';
+@import 'banner';
+@import 'intro';
+@import 'features';
+@import 'showcases';
+@import 'guide';
+@import 'topics';
+@import 'stats';
+@import 'faq';
+@import 'footer';
```

```suggestion
@import 'variables';
@import 'banner';
@import 'intro';
@import 'features';
@import 'showcases';
@import 'guide';
@import 'topics';
@import 'stats';
@import 'faq';
```
It would be better to extend the `main` css which is used in other pages than include the whole Bootstrap here. Also, some styles will be unnecessary after the change, please have it a look (here just an example).

---

`.scss` under `index/` is now imported to the main css. Also `masthead.scss`, `flash.scss` and `footer.scss` are merged into main css files.